app/assets/stylesheets/concepts.scss

```diff
@@ -157,3 +157,7 @@ div.synonym-change-request button {
   color: var(--gray-color);
   margin-top: 10px;
 }
+
+.concepts-mapping-count .d-flex{
```

Comment:
just don't use `d-flex`, instead directly use `d-inline` in your template/haml file.

Reply:
I can't do this cause it's inside the component's code.
It's safe like this

Reply:
The colors and padding is not good 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/0436ddc6-da6e-42f6-96de-b1c033f485ff)

`.concepts-mapping-count .d-flex` does not have any effect ?

Reply:
Tomorrow I'll see how to solve this on ur machine (it looks correct here and in Imad's machine)
![Capture d’écran du 2024-04-11 17-43-02](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/98e2b824-4d4f-4707-98b4-f001717aeb84)

![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/00000aa2-5633-48c3-81a0-18e8bf3d0b21)

Reply:
Maybe try in English not french mode

Reply:
and in E-PHY ontology

Reply:
same;
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/e2941b92-3a54-4b0e-8a89-72834f371123)

can you show me a screenshot of your inspect element css

Reply:
it seems to be fine now